packages/block-library/src/navigation/edit/test/navigation-menu-selector.js

```diff
@@ -271,6 +271,16 @@ describe( 'NavigationMenuSelector', () => {
 					} )
 				);
 
+				useNavigationMenu.mockReturnValue( {
+					navigationMenus: [],
+					hasResolvedNavigationMenus: false,
+					isResolvingNavigationMenus: true,
+					canUserCreateNavigationMenu: true,
+					canSwitchNavigationMenu: true,
+				} );
```

Comment:
Nit: I know this is not your fault, but these tests would be a lot easier to read if there were comments above these mocks explaining the state they create. For example: "Set menus to resolving"